lib/lambda_ethereum_consensus/fork_choice/handlers.ex

```diff
@@ -3,11 +3,80 @@ defmodule LambdaEthereumConsensus.ForkChoice.Handlers do
   Handlers that update the fork choice store.
```

Comment:
nit: would it make sense to move to handlers/on_tick.ex ?

Reply:
There are not that many functions, and some are shared with `on_block`, so we would need a `handlers/helpers.ex` too.